packages/orchestration/README.md

```diff
@@ -413,12 +415,12 @@ Both `chatCompletion()` and `getContent()` methods can throw errors.
 
 - **Axios Errors**:  
   When the chat completion request fails with a `400` status code, the caught error will be an `Axios` error.
-  The property `error.response.data.message` provides additional details about the failure.
+  Check `error.cause.response.data` for the error response body.
```

Comment:
[pp] Maybe instead of repeating this, we can just add it once as a tip/note

Reply:
Love this idea. Improved.